Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonButton.cs

```diff
@@ -268,65 +268,7 @@ public ButtonStyle ButtonStyle
 
         private void ResetButtonStyle() => ButtonStyle = ButtonStyle.Standalone;
 
-        [DefaultValue(false),
```

Comment:
Is this a "breaking change"? and needs to be documented

Reply:
Has now been put in `Readme`